ods-jekyll-theme.gemspec

```
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name          = "ods-jekyll-theme"
+  spec.version       = "1.0.0"
```

Since we won't really use this package in production, let's have a fake "0.0.0" version in case we need a real versioning like this in the future.

```suggestion
  spec.version       = "0.0.0"
```